sources/packages/backend/apps/api/src/route-controllers/student-account-applications/_test_/e2e/student-account-application.aest.controller.approveStudentAccountApplication.e2e-spec.ts

```diff
@@ -245,6 +245,73 @@ describe("StudentAccountApplicationAESTController(e2e)-approveStudentAccountAppl
     });
   });
 
+  it("Should create disbursement overawards when an exact match is found with matching last name, birth dates and SIN for importing a student record with disbursement overawards from SFAS.", async () => {
```

Comment:
This is a valid test and we can keep it, it is just not the code that would revert the deleted records.
The process would happen during the execution of the SFAS scheduler (sfas-integration.scheduler) during the [updateDisbursementOverawards](https://github.com/bcgov/SIMS/blob/5b9f09a6191dbd80be92cb005677e45d07e1983a/sources/packages/backend/libs/integrations/src/services/sfas/sfas-individual-import.service.ts#L144).
To create an E2E test, it needs to be under queue-consumers importing an SFAS bridge record for an individual with overawards data.

Reply:
@sh16011993 the current test is this tile is testing the API call and should stay where it is.
My comment, if not clear, is to have a second E2E test created for the queue-consumers.
legacy overawards are imported during student account creation and during the SFAS bridge process.